src/Membership/AdherentManager.php

```diff
@@ -2,25 +2,63 @@
 
 namespace AppBundle\Membership;
 
+use AppBundle\CitizenAction\CitizenActionManager;
+use AppBundle\CitizenInitiative\ActivitySubscriptionManager;
+use AppBundle\CitizenInitiative\CitizenInitiativeManager;
+use AppBundle\CitizenProject\CitizenProjectManager;
 use AppBundle\Collection\AdherentCollection;
+use AppBundle\Committee\CommitteeManager;
+use AppBundle\Committee\Feed\CommitteeFeedManager;
 use AppBundle\Entity\Adherent;
 use AppBundle\Entity\AdherentActivationToken;
 use AppBundle\Entity\CitizenInitiative;
 use AppBundle\Entity\BaseEvent;
+use AppBundle\Entity\Committee;
+use AppBundle\Entity\Summary;
+use AppBundle\Event\EventManager;
+use AppBundle\Event\EventRegistrationManager;
+use AppBundle\Report\ReportManager;
 use AppBundle\Repository\AdherentRepository;
 use Doctrine\Common\Persistence\ObjectManager;
 
 class AdherentManager
 {
     private $manager;
     private $repository;
+    private $citizenInitiativeManager;
+    private $citizenActionManager;
+    private $eventManager;
+    private $committeeManager;
+    private $committeeFeedManager;
+    private $activitySubscriptionManager;
+    private $citizenProjectManager;
+    private $reportManager;
+    private $registrationManager;
 
     public function __construct(
         AdherentRepository $repository,
-        ObjectManager $manager
+        ObjectManager $manager,
+        CitizenInitiativeManager $citizenInitiativeManager,
+        CitizenActionManager $citizenActionManager,
+        EventManager $eventManager,
+        CommitteeManager $committeeManager,
+        CommitteeFeedManager $committeeFeedManager,
+        ActivitySubscriptionManager $activitySubscriptionManager,
+        CitizenProjectManager $citizenProjectManager,
+        ReportManager $reportManager,
+        EventRegistrationManager $registrationManager
     ) {
         $this->repository = $repository;
         $this->manager = $manager;
+        $this->citizenInitiativeManager = $citizenInitiativeManager;
```

Comment:
All those dept for one method, I would create another service instead, something like `AdherentRegistry::unregister()`

Reply:
Done